CONTRIBUTING.md

```diff
@@ -69,168 +142,6 @@ or if using tox
-   You can do this by using the "Merge pull request" button on the release PR.
-
-6. Now you are ready to create the actual release. Navigate to the project page
-   on GitHub and draft a new release with the following settings:
-    * Tag version: `vX.Y.Z`
-    * Target: `main`
-    * Release title: `Release vX.Y.Z`
-    * Description: Includes a high-level list of things changed in this release.
-      Click `Publish release`.
-
-7. Merge the `release/vX.Y.Z` into `dev`.
-
-    ```bash
-    $ git checkout dev
-    $ git merge release/vX.Y.Z
-    $ git push
-    ```
-
-8. Finish off the tasks in the release checklist. Once everything is done, close
-   the release PR.
-
-9. Delete the release branch on GitHub.
-
-### Production hot fix
-
-A production hotfix is very similar to a full-scale release except that you do
-your work in a branch taken directly off of `main`. Hotfixes are useful in cases
-where you want to patch a bug in a released version, but `dev` has unreleased
-code in it already.
-
-**TBD: Insert diagram**
-
-1. Create a hot fix branch based off of `main`.
-
-   ```bash
-   $ git checkout main
-   $ git checkout -b hotfix/documentation-broken-links
-   $ git push --set-upstream origin hotfix/documentation-broken-links
-   ```
-
-2. Add a test case to validate the bug, fix the bug, and commit.
-
-   ```bash
-   ... add test, fix bug, verify
-   $ git add -A .
-   $ git commit -m "Fix broken links"
-   $ git push
-   ```
-
-3. Navigate to the project on [GitHub](https://github.com) and open a pull request
-   with the following branch settings:
-   * Base: `main`
-   * Compare: `hotfix/documentation-broken-links`
-     Paste your release checklist into the PR and work through the PR to get the
-     hotfix into production.
-
-4. At some point in the checklist you will merge the hotfix branch into `main`.
-   You can do this by using the "Merge pull request" button on the release PR.
-
-5. Now that the hotfix code is in `main` you are ready to create the actual
-   release. Navigate to the project page on GitHub and draft a new release with
-   the following settings:
-   * Tag version: `vX.Y.Z`
-   * Target: `main`
-   * Release title: `Release vX.Y.Z (hotfix)`
-   * Description: Includes a high-level list of things changed in this release.
-
-Click `Publish release`.
-
-*Note: Hotfix releases _are_ actual releases. You should bump at least the
-patch part of the version when releasing a hotfix, and so even hotfixes go
-through the process of creating a release like this.*
-
-1. Merge the `hotfix/documentation-broken-links` into `dev`.
-
-   ```bash
-   $ git checkout dev
-   $ git merge hotfix/documentation-broken-links
-   $ git push
-   ```
-
-2. Finish off the tasks in the release checklist. Once everything is done, close
-   the hotfix PR.
-
 ## Code Style
 
 We maintain a consistent code style throughout the project, so please ensure your changes align with our existing style.
```

Comment:
Maybe show examples of how people can run the automatic linting.

Reply:
That is offered in the above section